app/assets/stylesheets/components/_card_list.css.sass

```diff
@@ -3,7 +3,7 @@
 
   .Discussion,
```

Comment:
I can see this is not from this PR, but this breaks the `loose coupling` rule.
.CardList should not know about these two components